store/src/state.rs

```diff
@@ -444,16 +466,20 @@ impl State {
         Ok((account, nullifier_blocks))
     }
 
+    /// Lists known nullifiers with their inclusion blocks.
```

Comment:
For this, and the 2 method below, I wonder if we should say "Lists all known..." to emphasize that all elements are returned.

Also, should we add a note that these methods are intended primarily for testing?

Reply:
changed